src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -1368,7 +1368,7 @@ MulticopterAttitudeControl::start()
 	_control_task = px4_task_spawn_cmd("mc_att_control",
 					   SCHED_DEFAULT,
 					   SCHED_PRIORITY_MAX - 5,
-					   1700,
+					   2200,
```

Comment:
I did some preliminary investigation into the issue and it the stack frames generated by Clang were slightly larger. From what I've been reading, Clang's code size optimizations for ARM aren't as good as GCCs.